src/doc/en/developer/packaging.rst

```diff
@@ -412,19 +412,19 @@ begin with ``sdh_``, which stands for "Sage-distribution helper".
   may be given as arguments.
 
 - ``sdh_make [...]``: Runs ``$MAKE`` with the default target.
-   Additional arguments to ``$MAKE`` may be given as arguments.
+  Additional arguments to ``$MAKE`` may be given as arguments.
 
 - ``sdh_make_install [...]``: Runs ``$MAKE install`` with DESTDIR
-   correctly set to a temporary install directory, for staged
-   installations. Additional arguments to ``$MAKE`` may be given as
-   arguments. If ``$SAGE_DESTDIR`` is not set then the command is run
-   with ``$SAGE_SUDO``, if set.
+  correctly set to a temporary install directory, for staged
+  installations. Additional arguments to ``$MAKE`` may be given as
+  arguments. If ``$SAGE_DESTDIR`` is not set then the command is run
+  with ``$SAGE_SUDO``, if set.
 
 - ``sdh_setup_bdist_wheel [...]``: Runs ``setup.py bdist_wheel`` with
-   the given arguments, as well as additional default arguments used for
-   installing packages into Sage.
+  the given arguments, as well as additional default arguments used for
+  installing packages into Sage.
 
-- ``sdh_pip_install [...]``: The equivalent of running ``pip install``
+-  ``sdh_pip_install [...]``: The equivalent of running ``pip install``
    with the given arguments, as well as additional default arguments used for
```

Comment:
Two spaces, unlike the previous paragraph?

Reply:
Fixed in 6731a1c